muzlag/internal/cmd/bot/bot.go

```diff
@@ -0,0 +1,72 @@
+package bot
+
+import (
+	"fmt"
+	"log"
+	"log/slog"
+
+	"github.com/bwmarrin/discordgo"
+
+	"github.com/Drozd0f/gobots/muzlag/internal/bot"
+	"github.com/Drozd0f/gobots/muzlag/internal/bot/handlers"
+	"github.com/Drozd0f/gobots/muzlag/internal/config"
+	"github.com/Drozd0f/gobots/muzlag/internal/service/player"
+	"github.com/Drozd0f/gobots/muzlag/pkg/ffmpeg"
+	pkgLog "github.com/Drozd0f/gobots/muzlag/pkg/log"
+	"github.com/Drozd0f/gobots/muzlag/pkg/ytdl"
+)
+
+func NewBot() *discordgo.Session {
```

Comment:
https://go.dev/doc/modules/layout

packages that bellongs to cmd is not for this kind of stuff.
CMD is root directory of all subprograms that can be executed, for examble

/cmd/bot/main.go - this main file will setup and run discord bot
/cmd/server/main.go - this main file will setup http server and run it
/cmd/migrate/main.go - this main file is responsible for migrations, and you can make something like that

go build -o /bin/bot ./cmd/bot - it will return binary for your bot, and so on

![image](https://github.com/user-attachments/assets/81a578d0-16f7-424d-ada8-0507a11fb348)